README.md

```diff
@@ -98,9 +102,9 @@ We should also consider mob programming sometimes. This can be very good for spi
 
 ## How to help more generally:
 
-    Even if junior, you can read PRs. This is also work. Even if it’s just to say good job, or that you don’t understand something and could it be made clearer. Reading and responding to PRs is a skill, and one that I’m still learning. Dive in, or if you are worried about taking steps, ask someone to give you a hand. 
-    We love love love people who contribute to the README. If you document your struggles as you go through, these can be excellent elements to add to the README or form the basis for a FAQ. This helps others in a big way.
-    Get involved on Slack where you can.
+    - Even if junior, you can read PRs. This is also work. Even if it’s just to say good job, or that you don’t understand something and could it be made clearer. Reading and responding to PRs is a skill, and one that I’m still learning. Dive in, or if you are worried about taking steps, ask someone to give you a hand. 
+    - We love love love people who contribute to the README. If you document your struggles as you go through, these can be excellent elements to add to the README or form the basis for a FAQ. This helps others in a big way.
+    - Get involved on Slack where you can.
```

Comment:
Please remove the indentation in each bullet point.

```suggestion
- Even if junior, you can read PRs. This is also work. Even if it’s just to say good job, or that you don’t understand something and could it be made clearer. Reading and responding to PRs is a skill, and one that I’m still learning. Dive in, or if you are worried about taking steps, ask someone to give you a hand. 
- We love love love people who contribute to the README. If you document your struggles as you go through, these can be excellent elements to add to the README or form the basis for a FAQ. This helps others in a big way.
- Get involved on Slack where you can.
```

Reply:
Done.